python/hudi/_internal.pyi

```diff
@@ -38,7 +38,8 @@ class HudiTable:
     def __init__(
         self,
         table_uri: str,
-        options: Optional[Dict[str, str]] = None,
+        hudi_options: Optional[Dict[str, str]] = None,
+        storage_options: Optional[Dict[str, str]] = None,
```

Comment:
you can revert this change - we keep python and rust api as close as possible

Reply:
will do, I'll introduce HudiTableBuilder in python bindings instead